packages/@aws-cdk/aws-eks/README.md

```diff
@@ -526,26 +526,33 @@ With services account you can provide Kubernetes Pods access to AWS resources.
 
 ```ts
 // add service account
-const serviceAccount = cluster.addServiceAccount('MyServiceAccount');
+const sa = cluster.addServiceAccount('MyServiceAccount', {});
```

Comment:
The `{}` should not be needed. If it is, it's a bug.

```suggestion
const sa = cluster.addServiceAccount('MyServiceAccount');
```

Reply:
This has been fixed with in PR #8084 and should be removed.

Reply:
OK. Removed.